fuzzers/structure_aware/baby_fuzzer_grimoire/src/main.rs

```diff
@@ -83,6 +83,7 @@ pub fn main() {
     };
 
     // Create an observation channel using the signals map
+    #[allow(static_mut_refs)] // only a problem in nightly
```

Comment:
We should not use static mut refs here. 
See this discussion as reference
https://github.com/AFLplusplus/LibAFL/pull/2770#discussion_r1885707774